src/Modules/CalcSections.lua

```diff
@@ -951,105 +951,149 @@ return {
 } }
 } },
 { 1, "EleAilments", 1, colorCodes.OFFENCE, {{ defaultCollapsed = false, label = "Elemental Ailments", data = {
```

Comment:
Just since we have dedicated panes for Ignite and Poison.  I found myself looking for Ignite once under this section

```suggestion
{ 1, "EleAilments", 1, colorCodes.OFFENCE, {{ defaultCollapsed = false, label = "Non-Damaging Ailments", data = {
```